mina.go

```diff
@@ -116,6 +117,18 @@ func (m *Mina) ServeHTTP(wr http.ResponseWriter, req *http.Request) {
 	req.Header.Del("If-Modified-Since")
 	req.Header.Del("If-None-Match")
 
+	if req.Header.Get(XHeaderName) == XHeaderValueIgnore {
```

Comment:
Cool! Let's use a different header name for requests (e.g., RequestOptionsHeaderName being "X-MINA-OPTIONS"), and leave XHeaderName for the response header.

If you want, you could rename XHeaderName to something better as well, e.g., ResponseCacheHeaderName. :game_die: